statesync/stateprovider.go

```diff
@@ -96,6 +97,18 @@ func NewLightClientStateProvider(
 	}, nil
 }
 
+func uniqServers(servers []string) []string {
```

Comment:
nit, it seems to me that it could be a generic function such as `UniqueString()` since Golang doesn't have a function like Ruby's `uniq()`, which could be implemented in `libs/strings/string.go`.

Reply:
I agree with you, but I don't make a generic function at this time. If we need to use it, refactoring is better at that time.